lib/customer.rb

```diff
@@ -0,0 +1,49 @@
+# require "awesome_print"
+# require_relative '../data/csv_practice'
+require "csv"
+
+# class that creates an instance of a customer
+class Customer
+
+  # creates methods that will allow reading or reading/writing of specified instance variables
+  attr_reader :id
+  attr_accessor :email, :address
```

Comment:
The email and address fields don't need to be accessors.  You want to be careful before granting a user the ability to directly change your instance variables.